R/bed_absdist.r

```diff
@@ -41,32 +45,41 @@
 
 bed_absdist <- function(x, y, genome) {
   
-  x <- group_by(x, chrom, add = TRUE)
-  y <- group_by(y, chrom, add = TRUE)
-  
-  # make sure that y tbl has same grouping as x tbl
-  y <- set_groups(y, x)
+  # find minimum shared groups
+  xy_groups <- shared_groups(y, x)
   
+  x <- group_by_(x, .dots = c("chrom", xy_groups))
```

Comment:
Surprisingly, this actually works because `c("chrom", xy_groups)` returns a `list` object, which is a valid input type for the`.dots` parameter. 

```r
c("chrom" , list("start", "end"))
#> [[1]]
#> [1] "chrom"
#> 
#> [[2]]
#> [1] "start"
#> 
#> [[3]]
#> [1] "end"
```